libs/sdk-react-native/ios/BreezSDKMapper.swift

```diff
@@ -211,6 +222,20 @@ class BreezSDKMapper {
         }
     }
     
+    static func asLnUrlWithdrawRequestData(reqData: [String: Any]) -> LnUrlWithdrawRequestData? {
+        if let callback = reqData["callback"] as? String,
+            let k1 = reqData["k1"] as? String,
+            let defaultDescription = reqData["defaultDescription"] as? String,
+            let minWithdrawableStr = reqData["minWithdrawable"] as? String,
+            let maxWithdrawableStr = reqData["maxWithdrawable"] as? String {
+            if let minWithdrawable = UInt64(minWithdrawableStr), let maxWithdrawable = UInt64(maxWithdrawableStr) {
+                return LnUrlWithdrawRequestData(callback: callback, k1: k1, defaultDescription: defaultDescription, minWithdrawable: minWithdrawable, maxWithdrawable: maxWithdrawable)
```

Comment:
Is there a more elegant way to do this?

Reply:
Are you referring to the type conversion from js to swift? shouldn't we read it directly as double?

Reply:
The int64 values are converted to string over the bridge, but i'll refactor that.